decidim-participatory_processes/app/views/decidim/participatory_processes/participatory_processes/show.html.erb

```diff
@@ -104,6 +104,7 @@ edit_link(
   </div>
   <% if current_participatory_space.show_statistics? %>
     <%= render partial: "statistics" %>
+    <%= render partial: "metrics" %>
```

Comment:
Doesn't this partial make the statistics one deprecated? I'd remove it (not just the line, the whole file/feature)

Reply:
Do you mean to remove all configurations/manifest declaration for `ParticipatoryProcess`? Or only the lines to load and show them in this page?

Reply:
@decidim/product I need you to confirm this modification in `ParticipatoryProcess` show page

Reply:
Sorry @MarcReniu I'm not sure I understand. What is the modification we need to confirm?

Reply:
The modification is to remove the statistics showing in `ParticipatoryProcess` and just keave metrics charts

Reply:
@decidim/product I need to unlock this thread as soon as possible

PS: I created a new PR at #4455 to focus on ParticipatoryProcess home

Reply:
In a talk with @carolromero, we agreed to leave statistics as it is, by now